woocommerce/woocommerce
# Add product management SKU

### All Submissions:






### Changes proposed in this Pull Request:

Adds the SKU field and inventory section to the new product management experience.

Closes #34386 .
<img width="337" alt="Screen Shot 2022-10-06 at 7 12 24 AM" src="https://user-images.githubusercontent.com/10561050/194335819-da8591c0-489a-460f-a94f-05333dbad3cd.png">
<img width="489" alt="Screen Shot 2022-10-06 at 7 12 10 AM" src="https://user-images.githubusercontent.com/10561050/194335822-9d0fd030-7a19-4c7c-97b0-981f28920b75.png">
<img width="281" alt="Screen Shot 2022-10-06 at 7 11 59 AM" src="https://user-images.githubusercontent.com/10561050/194335823-865d048c-a1c7-4d29-88f7-e27880e645ae.png">



### How to test the changes in this Pull Request:

1. Visit the new product page `Products -> Add new (MVP)`
2. Type in a product name and blur the input
3. Note the SKU has been added in slug format to the SKU field
4. Change the SKU to a bad format including non-alphanumeric characters and spaces
5. Note the error
6. Make a very long SKU (greater than 160 characters) and note the error
7. Save and make sure the SKU persists on refresh
8. Remove the product name so it's empty
9. Blur the field and make sure the error appears

### Other information:




### FOR PR REVIEWER ONLY:
